pkgs/top-level/python-packages.nix

```
@@ -817,10 +819,16 @@ in {
 
   langdetect = callPackage ../development/python-modules/langdetect { };
 
+  launchpadlib = callPackage ../development/python-modules/launchpadlib { };
+
   lazr_config = callPackage ../development/python-modules/lazr/config.nix { };
 
   lazr_delegates = callPackage ../development/python-modules/lazr/delegates.nix { };
 
+  lazr_restfulclient = callPackage ../development/python-modules/lazr/restfulclient.nix { };
```

```suggestion
  lazr-restfulclient = callPackage ../development/python-modules/lazr-restfulclient/default.nix { };
```
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines for the normalization.

Since each of these correspond to a package on pypi, they should be treated like individual packages. The fact that they share a namespace is just coincidental.

---

Fixed, thanks!